tools/autograd/templates/Functions.cpp

```diff
@@ -417,7 +417,7 @@ Tensor softplus_double_backward(const Tensor & grad, const Tensor & input, Scala
 
 Tensor as_strided_backward(const Tensor & grad, TensorGeometry base, IntList sizes, IntList strides, int64_t storage_offset) {
   auto src = base.zeros_with_stride(grad.type());
-  src.as_strided(sizes, strides, storage_offset - base.storage_offset).copy_(grad);
+  src.as_strided(sizes, strides, storage_offset - base.storage_offset()).copy_(grad);
```

Comment:
!! How did that even work before

Reply:
This patchset contains a change to `storage_offset` in `TensorGeometry` to convert it into a method